ehitchhiking-ui/src/app/app.module.ts

```diff
@@ -24,11 +25,12 @@ import {RequestCache} from '@shared/services/request.cache.service';
 		HttpClientModule,
 		MatNativeDateModule,
 	],
-  providers: [
-    ApiService,
-    {provide: HTTP_INTERCEPTORS, useClass: CachingInterceptor, multi: true},
-    {provide: RequestCache, useClass: RequestCache},
-  ],
+	providers: [
+		ApiService,
+		BlackListApiService,
```

Comment:
You should not add BlackListApiService as provider here. BlackListApiService is required in BlackListModule

Reply:
done